.github/workflows/build.yml

```diff
@@ -13,12 +13,30 @@ jobs:
       image: quay.io/centos/centos:stream9
 
     steps:
-    - uses: actions/checkout@v3
+    - name: Install dependencies
+      run: |
+        dnf install \
+                autoconf \
```

Comment:
Minor: We can remove `autoconf` and `automake` for now, I think.

Reply:
Fixed